revocation/crl/bundle.go

```diff
@@ -0,0 +1,26 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crl
+
+import (
+	"crypto/x509"
+)
+
+// Bundle is in memory representation of the Bundle tarball, including base CRL.
```

Comment:
I'm trying to understand what does this mean 'Bundle is in memory representation of the Bundle tarball'. Looks like `Bundle` is the actual record been stored in the cache? If yes, then this should be moved into `cache.go` file?

Reply:
Updated the comment. Please have a look again. It just a collection of CRLs now.